source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,71 @@
 ---
-title: Set up 3D Secure
+title: Set up Strong Customer Authentication
 last_reviewed_on: 2020-06-10
 review_in: 3 months
 weight: 170
 ---
 
-# Set up 3D Secure (3DS)
+# Strong Customer Authentication
 
-How you set up 3DS payment authentication depends on:
+Strong Customer Authentication (SCA) makes online payments more secure for users and reduces fraud. SCA is part of the Payment Service Directive (PSD2).
 
-- which Payment Service Provider (PSP) you're using
-- if you want to set up 3D Secure 2 (3DS2), which adds an extra layer of identity confirmation before your user pays
+You must implement SCA by:
+- 31 December 2020 if you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein
+- 14 September 2021 if you only take payments from within the UK
 
-## Stripe
+To implement SCA, you must set up 3D Secure 2 (3DS2) or 3D Secure 1 (3DS1) payment authentication.
 
-You do not need to do anything, because we set up 3DS2 on your service by default.
+How you set up 3DS2 or 3DS1 payment authentication depends on which Payment Service Provider (PSP) you’re using.
 
-## Worldpay
+## Set up SCA with Stripe
 
-### Set up 3DS
+You do not need to do anything because we set up 3DS2 on your service by default.
 
-1. Ask your Worldpay account manager to set up 3DS on your merchant code for all payments.
-2. Once Worldpay have set up your merchant code, select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-3. In the settings, select the __View__ link under __3D Secure__.
-4. Select __On__.
+Stripe automatically applies exemptions.
+
+## Set up SCA with Worldpay
+
+Worldpay sends all transactions using 3DS2 or 3DS1. 3DS Flex is the Worldpay product that enables 3DS2 payment authentication. You must pay a fee to Worldpay to use 3DS Flex.
+
+If you are a Government Banking customer, Government Banking sends you your 3DS Flex credentials and tells you to enable 3DS Flex on your account.
+
+If you have a direct contract with Worldpay, Worldpay sends you your 3DS Flex credentials.
+
+Worldpay does not apply any exemptions. Contact GOV.UK Pay to apply for exemptions for your transactions.
```

Comment:
can we add something about digital wallets here? 

digital wallets meet the criteria of SCA so no additional check is needed. also we don't need to get into costs here but it does make it cheaper for services.